bigframes/session/__init__.py

```diff
@@ -1170,7 +1170,11 @@ def _read_csv_w_bigquery_engine(
 
         table_id = self._loader.load_file(filepath_or_buffer, job_config=job_config)
         df = self._loader.read_gbq_table(
-            table_id, index_col=index_col, columns=columns, names=names
+            table_id,
+            index_col=index_col,
+            columns=columns,
+            names=names,
+            is_index_in_columns=True,
```

Comment:
I'm a bit confused by this parameter name. Wouldn't the `read_gbq_table` function be able to figure out that the index columns are present already?